update toc.yml to update MSDN forum link to Azure LBS link.

jierong opened the pull request:
update MSDN forum link to https://social.msdn.microsoft.com/Forums/en-US/home?forum=azurelbs

azurecla commented:
Hi __@cdavid__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@cdavid__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

lizap commented:
I talked with Catalin and added these changes to an open PR in the private repo. Closing this PR in favor of this one: https://github.com/Azure/azure-content-pr/pull/15020

BryanLa commented:
@lizap - I noticed this is still open; LMK if you would like me to close.

BryanLa commented:
@lizap - went ahead and closed this one, per your recommendation (in addition to the fact that it's in a merge conflict state now as well)

jierong committed:
update toc.yml to update MSDN forum link to Azure LBS link.

update MSDN forum link to https://social.msdn.microsoft.com/Forums/en-US/home?forum=azurelbs

dominicbetts commented on articles/location-based-services/toc.yml:
```diff
@@ -46,7 +46,7 @@
     href: https://azure.microsoft.com/roadmap/?category=internet-of-things
   - name: MSDN forum
-    href: https://social.msdn.microsoft.com/Forums/azure
+    href: https://social.msdn.microsoft.com/Forums/en-US/home?forum=azurelbs
```
Can you update your PR to use the following URL (without the locale identifier):
`https://social.msdn.microsoft.com/Forums/home?forum=azurelbs`
Thanks!

kgremban replied:
@dominicbetts For social.msdn.microsoft forums, the en-US locale is required for the URL to render correctly. When the locale isn't included, the URL drops the ?forum=azurelbs.

jierong replied:
@dominicbetts , @kgremban, I think I found a way to remove locale info by adding azure in the URL, and it will be:

https://social.msdn.microsoft.com/Forums/azure/home?forum=azurelbs

dominicbetts reviewed (changes requested):
Hi @jierong - thanks for you contribution. Pleas make the suggested change to your PR - we can then get it merged. Thanks!

jierong committed:
Update toc.yml
